deploy/Chart/templates/controller/webhook.yaml

```diff
@@ -32,7 +33,7 @@ webhooks:
 - admissionReviewVersions:
   - v1
   clientConfig:
-    caBundle: {{ b64enc $ca.Cert }}
+    caBundle: {{ $validatingWebhookCaBundle }}
```

Comment:
The missing piece was that we were always using a new caBundle here. If the secret stays the same and pod gets recreated, we shouldn't create a new caBundle for this. That creates all the failures of second time test runs.

Also, ucp does it this way: https://github.com/radius-project/radius/blob/main/deploy/Chart/templates/ucp/apiservice.yaml#L42.

cc/ @youngbupark @rynowak @lakshmimsft

Reply:
Can you please open new PR for helm update? this is the actual fix for upgrading scenario. So it is better to create new pr for this.